src/events/callback/handler.cpp

```diff
@@ -11,6 +11,6 @@ namespace Bald {
     Handler::~Handler() {
         auto iter = std::find(m_TakenID.begin(), m_TakenID.end(), m_ID);
         m_TakenID.erase(iter);
+        CORE_LOG_INFO("[Handler] Successfully removed current ID");
     }
-
```

Comment:
We always keep one new line before namespace ends. You can change this for now, but hopefully, we will set up a normalized code style for our project.

Reply:
"hopefully" kappa